app/controllers/api/docs_controller.rb

```diff
@@ -0,0 +1,10 @@
+module API
+  class DocsController < BaseController
+    skip_before_action :authenticate_user!
+    before_action :authenticate_user
```

Comment:
Do we need this?

Reply:
Yes, as docs are viewable by non-logged-in users.